gcloud-java-resourcemanager/src/main/java/com/google/gcloud/resourcemanager/testing/package-info.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * A testing helper for Google Cloud Resource Manager.
+ *
+ * <p>A simple usage example:
+ * <p>Before the test:
+ * <pre> {@code
+ * LocalResourceManagerHelper resourceManagerHelper = LocalResourceManagerHelper.create();
+ * // TODO(ajaykannan): implement the following line when ResourceManagerImpl is checked in.
```

Comment:
I am not sure these TODOs are relevant in this file but if you decide to keep them make sure we fix that later.